src/devtools/client/debugger/src/utils/prefs.js

```diff
@@ -158,17 +158,14 @@ export const features = new PrefsHelper("devtools.debugger.features", {
   jumpLine: ["Bool", "jump-line"],
 });
 
-export const asyncStore = asyncStoreHelper("debugger", {
-  pendingBreakpoints: ["pending-breakpoints", {}],
```

Comment:
:hand: This seems like an issue - do we actually still persist these anywhere?

Reply:
Yes. See all of the other replies 😆